MAUI/TreeView/Data-population.md

```diff
@@ -0,0 +1,306 @@
+         stone,
+         wind
+      };
+      
+      camera.SubFiles = new ObservableCollection<FileManager>
+      {
+         img0,
+         img1
+      };
+
+      video.SubFiles = new ObservableCollection<FileManager>
+      {
+         video1,
+         video2
+      };
+
+      nodeImageInfo.Add(doc);
+      nodeImageInfo.Add(download);
+      nodeImageInfo.Add(mp3);
+      nodeImageInfo.Add(pictures);
+      nodeImageInfo.Add(video);
+      imageNodeInfo = nodeImageInfo;
+  }
+}
+{% endhighlight %}
+{% endtabs %}
+
+### Bind to hierarchical datasource
+
+To create a tree view using data binding, set a hierarchical data collection to the `ItemsSource` property. And set the child object name to the `ChildPropertyName` property.
+
+{% tabs %}
+{% highlight xaml %}
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.TreeView;assembly=Syncfusion.Maui.TreeView"
+             xmlns:treeviewengine="clr-namespace:Syncfusion.TreeView.Engine;assembly=Syncfusion.Maui.TreeView"
+             x:Class="GettingStarted.MainPage">
+    <ContentPage.BindingContext>
+       <local:FileManagerViewModel x:Name="viewModel"></local:FileManagerViewModel>
+    </ContentPage.BindingContext>
+    <ContentPage.Content>
+       <syncfusion:SfTreeView x:Name="treeView"
+                              ChildPropertyName="SubFiles"
+                              ItemsSource="{Binding ImageNodeInfo}"/>
+       </syncfusion:SfTreeView>
+    </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfTreeView treeView = new SfTreeView();
+FileManagerViewModel viewModel = new FileManagerViewModel ();
+treeView.ChildPropertyName = "SubFiles";
+treeView.ItemsSource = viewModel.ImageNodeInfo; 
+MainPage = new ContentPage { Content = treeView };
+
+{% endhighlight %}
+{% endtabs %}
+
+Download the entire source code from GitHub here.
+
+![.NET MAUI TreeView in bound mode](TreeView_images/TreeViewBound.png)
+
+## Populating Nodes without data binding - Unbound Mode
+
+You can create and manage the `TreeViewNode` objects by yourself to display the data in a hierarchical view. Create the node hierarchy by adding one or more root nodes to the `Nodes` collection. Each `TreeViewNode` can then have more nodes added to its Children collection. You can nest the tree view nodes to any depth you need. By using the `Content` property, you can get the data object associated with the tree view node. For the Bound mode, you can bind the data object using the `Content` property by setting the `ItemTemplateContextType` to `Node`. For Unbound mode, you can directly set the `Content` value. You can get the parent node of the treeview nodes by using the `ParentNode` property of the `TreeViewNode`. By using the `Level` property of the TreeViewNode, you can get the zero-based depth of the tree node in the TreeView control. For the `Level` property, the root node is considered the first level of nesting and returns 0.

```

Comment:
You can create and manage the `TreeViewNode` objects by yourself to display the data in a hierarchical view. Create the node hierarchy by adding one or more root nodes to the `Nodes` collection. Each `TreeViewNode` can then have more nodes added to its Children collection. You can nest the tree view nodes to any depth you need. By using the `Content` property, you can get the data object associated with the tree view node. For the Bound node, you can bind the data object using the `Content` property by setting the `ItemTemplateContextType` to `Node`. For Unbound node, you can directly set the `Content` value. You can get the parent node of the treeview nodes by using the `ParentNode` property of the `TreeViewNode`. By using the `Level` property of the TreeViewNode, you can get the zero-based depth of the tree node in the TreeView control. For the `Level` property, the root node is considered the first level of nesting and returns 0.